content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -12,16 +12,17 @@ Scan your Web Applications to discover external vulnerabilities.
 
 The complexity of today's applications, combined with the constant evolution of cyber threats, makes it increasingly difficult to ensure security measures. Traditional methods of security assessment often fall short of providing continuous insights, leading to potential blind spots.
 
-Scans address these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
+DAST Scanner addresses these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
 
 The beta period runs until December 31, 2023. During the beta, you have access to three scans a month, allowing you to fully explore this new feature. Scans reset on the first of the month. Anticipate usage and limitation adjustments in 2024.
 
-To access the Scans Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
+To access the DAST Scanner Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
 
 We use the following IP Address to make requests to your target: **52.19.40.38**
 
-## How to Use Scans
+## How to Use DAST Scanner
 
+<!-- TODO: update this video -->
 [Watch the how-to video here.](https://videos.cobalt.io/watch/4D8kQcTmPBE3HdWy6X3fHW)
```

Comment:
I noticed that the demo video we have will also need to be updated to show the recent UX.